src/components/Square.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
   // For Wave 1 enable this 
   //  Component to alert a parent 
   //  component when it's clicked on.
+  const clickSquare = () =>{
+    props.onClickCallback(props.id);
+  };
```

Comment:
Nice click handler that allow you to call the passed in method along with the id from props.